test/karma.conf.js

```diff
@@ -146,7 +144,5 @@ module.exports = config => {
         // Continuous Integration mode
         // if true, Karma captures browsers, runs the tests and exits
         singleRun: false,
-
-        browserify: {},
```

Comment:
Note: these changes weren't strictly necessary, but bring the config more in line with enketo-express.